pyuvdata/tests/test_ms.py

```diff
@@ -77,6 +64,24 @@ def test_readMSreadUVFITS():
     # them to be the same anyways.
     ms_uv.history = ""
     uvfits_uv.history = ""
+
+    # the objects won't be equal because uvfits adds some optional parameters
+    nt.assert_false(uvfits_uv == ms_uv)
+    # they are equal if only required parameters are checked:
+    nt.assert_true(uvfits_uv.__eq__(ms_uv, check_extra=False))
+
+    # set those parameters to none to check that the rest of the objects match
+    for p in uvfits_uv.extra():
```

Comment:
Actually, as I think about it more, I don't think we should, mostly because antenna_positions are technically not required and I think you usually wouldn't want to lose those. So I think it's better to let people clear things as needed.